__tests__/index.test.js

```diff
@@ -31,3 +31,36 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("status 200: responds with an object containing all endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        for (let key in body.endpoints) {
```

Comment:
Before any iteration in a test, we should check the array's length to be sure that we have something to iterate over